VTEX - Marketplace Protocol.json

```diff
@@ -1262,14 +1262,15 @@
                         }
                     },
                     {
-                        "name": "orderId",
+                        "name": "sellerOrderId",
                         "in": "path",
-                        "description": "ID of the order being fulfilled",
+                        "description": "ID of the seller notified to start the fulfillment process. It can be a VTEX seller or an external seller.",
```

Comment:
ID of the **seller**, ou ID do **pedido do seller**?

Reply:
```suggestion
                        "description": "Order ID in the seller that was notified to start the fulfillment process. It can be a VTEX seller or an external seller.",
```